packages/amplify-graphql-model-transformer/src/graphql-model-transformer.ts

```diff
@@ -347,6 +366,7 @@ export class ModelTransformer extends TransformerModelBase implements Transforme
     type: ObjectTypeDefinitionNode,
     typeName: string,
     fieldName: string,
+    resolverLogicalId: string,
```

Comment:
could we compute the logical ID of the resolver inside these functions using the typeName and fieldName rather than passing it in? Seems like that would significantly reduce the number of lines of code that need to change

Reply:
typeName in the addResolver is either Query/Mutation/Sync, not the actual object type

Reply:
also fieldName parameter here is already altered (aka: op + Field / or user custom query name) to match the V1 logic I have no data here

Reply:
ah I see. Okay I guess that makes sense. Just seems like this requires updates in a lot of places and if we could cut that down at all that would be preferable

Reply:
open for suggestions :) wasn't my first approach, this was the least amount of changes so far